src/PythonMigrationViewExtension/IronPythonInfoDialog.xaml.cs

```diff
@@ -68,6 +68,14 @@ private void OnMoreInformationButtonClicked(object sender, RoutedEventArgs e)
             this.Close();
         }
 
+        private void ToggleIronPythonDialog(object sender, RoutedEventArgs e)
+        {
+            if(DisableIronPythonDialogCheck.IsChecked.HasValue && DisableIronPythonDialogCheck.IsChecked.Value)
+                ViewModel.DynamoViewModel.IsIronPythonDialogDisabled = true;
+            else
+                ViewModel.DynamoViewModel.IsIronPythonDialogDisabled = false;
```

Comment:
Could it be null? If not, how about
```
ViewModel.DynamoViewModel.IsIronPythonDialogDisabled = DisableIronPythonDialogCheck.IsChecked;
```

Reply:
keeping the if in place, just in case

Reply:
With the new code, it seems we can never update the value when `DisableIronPythonDialogCheck.IsChecked.Value` is `false`? Can you verify?

Reply:
Ah.. sorry, fixed